dev/com.ibm.ws.jdbc_fat_driver/publish/servers/com.ibm.ws.jdbc.fat.driver/server.xml

```diff
@@ -37,6 +37,12 @@
       <containerAuthData user="dbuser1" password="{xor}Oz0vKDtu"/>
     </dataSource>
     
+    <dataSource id="fatDriverInvalidURLLoginTimeout" jndiName="jdbc/fatDriverInvalidURLLoginTimeout">
+      <jdbcDriver libraryRef="FATDriverLib" internal.nonship.function="This is for internal development only. Never use this in production"/>
+      <properties url="jdbc:fatdriver:memory:jdbcdriver1;create=true;LoginTimeout=10" user="dbuser2" password="{xor}Oz0vKDtt"/>
+      <containerAuthData user="dbuser1" password="{xor}Oz0vKDtu"/>
```

Comment:
Given that this data source is only meant to test loginTimeout, why the complexity with different users?